doc/news/changes.h

```diff
@@ -110,7 +110,17 @@ inconvenience this causes.
 <a name="specific"></a>
 <h3>Specific improvements</h3>
 
-<ol>
+<ol> 
+ <li> New: Add remove_hanging_nodes and remove_anisotropy in
+ GridTools. remove_hanging_nodes detects cells with hanging nodes and
+ refines the neighbours in the direction that removes hanging nodes or
+ in everery directions. remove_anisotropy refines a mesh
+ anisotropically such that the resulting mesh is composed by cells
```

Comment:
maybe `until the resulting mesh is composed by cells with ratio between the extension in each coordinate direction lower than a fixed value`.